webots_ros2_epuck/webots_ros2_epuck/epuck_node.py

```diff
@@ -46,47 +42,33 @@
 ]
 
 
-DEVICE_CONFIG = {
-    'camera': {'topic_name': ''},
-    'robot': {'publish_base_footprint': True},
-    'ps0': {'always_publish': True},
-    'ps1': {'always_publish': True},
-    'ps2': {'always_publish': True},
-    'ps3': {'always_publish': True},
-    'ps4': {'always_publish': True},
-    'ps5': {'always_publish': True},
-    'ps6': {'always_publish': True},
-    'ps7': {'always_publish': True},
-    'tof': {'always_publish': True}
-}
-
-
-class EPuckDriver(WebotsDifferentialDriveNode):
-    def __init__(self, args):
-        super().__init__(
-            'epuck_driver',
-            args,
-            wheel_distance=DEFAULT_WHEEL_DISTANCE,
-            wheel_radius=DEFAULT_WHEEL_RADIUS
-        )
-        self.start_device_manager(DEVICE_CONFIG)
+class EPuckNode(Node):
+    def __init__(self):
+        super().__init__('epuck_node')
+        self.get_logger().info("Epuck node has been started.")
 
         # Intialize distance sensors for LaserScan topic
-        self.distance_sensors = {}
+        self.__subscriber_dist_sensors = {}
+        self.__dists = {}
         for i in range(NB_INFRARED_SENSORS):
-            sensor = self.robot.getDistanceSensor('ps{}'.format(i))
-            sensor.enable(self.timestep)
-            self.distance_sensors['ps{}'.format(i)] = sensor
+            self.__dists['ps{}'.format(i)] = OUT_OF_RANGE
+
+        self.__subscriber_dist_sensors['ps0'] = self.create_subscription(Range, '/e_puck/ps0', self.__process_dist_sens_0, 1)
```

Comment:
```suggestion
        for i in range(7):
            self.__subscriber_dist_sensors[f'ps{i}'] = self.create_subscription(Range, f'/e_puck/ps{i}', lambda message: self.__on_distance_sensor_message(message, i), 1)
```
I believe something like this is possible (maybe slightly different)

Reply:
Thanks, I had no idea if it was possible to configure the name of the callback